### PR TITLE
Task 68 fix code issues

### PR DESCRIPTION
**Pull request by melonholy**

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  ITechYoutube: 'https://www.youtube.com/user/iTechArt',
+  ITechFacebook: 'https://www.facebook.com/iTechArt.Group',
+  linkedInURL: 'api/users/auth/linkedin',
+  googleURL: 'api/users/auth/google',
+  resetRoute: 'users/reset',
+  changePasswordRoute: 'users/change-password',
+  loginRoute: 'users/login',
+  registrationRoute: 'users/registration',
+  publicProfileRoute: 'profile/public',
+  Twitter:'http://twitter.com/',
```
Comment (universezzz) [inline src/utils/constants.js]: code style my sun <3 <3 <3

File: src/containers/profile/info/index.jsx [hunk 2]
```diff
@@ -6,15 +6,14 @@ import { FaFacebookF, FaLinkedinIn, FaTwitter } from 'react-icons/fa';
...
-import { Profile } from '../profile';
+import { getProfileRequest, updateProfileRequest } from '../../../actions/profile';
+import { links } from "../../../utils/constants";
+import  { Profile }  from '../profile';
 
 import styles from './styles.module.scss';
 
-const ProfileTab = ({ profile, getProfileRequest, updateProfileRequest }) => {
+const Info = ({ profile, getProfileRequest, updateProfileRequest }) => {
+
```
Comment (slavasgod) [inline src/containers/profile/info/index.jsx]: Change naming. What means "Info"?

File: src/services/auth.js [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
 
-import setAuthToken from './setAuthToken';
 import { httpService } from './httpService';
+import { links } from '../utils/constants';
 
 const config = {
   headers: {
     'Content-Type': 'application/json',
-    'Authorization' : localStorage.token,
+    Authorization: localStorage.token,
```
Comment (universezzz) [inline src/services/auth.js]: It should be deleted

**Review by slavasgod — CHANGES_REQUESTED** [review]

**Review by universezzz — CHANGES_REQUESTED** [review]

File: src/utils/constants.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+export const links = {
+  iTechVK: 'https://vk.com/itechart.group',
+  iTechYoutube: 'https://www.youtube.com/user/iTechArt',
+  iTechFacebook: 'https://www.facebook.com/iTechArt.Group',
+  inkedInURL: 'api/users/auth/linkedin',
```
Comment (universezzz) [inline src/utils/constants.js]: What's inkedInURL? Maybe you mean that you're sucker?

**Review by universezzz — CHANGES_REQUESTED** [review]

**Review by slavasgod — APPROVED** [review]

**Review by universezzz — APPROVED** [review]

**Pull request merged**